apis/betterself/v1/supplements/serializers.py

```diff
@@ -56,20 +56,32 @@ class IngredientCompositionReadOnlySerializer(serializers.Serializer):
 
 class IngredientCompositionCreateSerializer(serializers.Serializer):
     ingredient_uuid = serializers.UUIDField(source='ingredient.uuid')
-    measurement_uuid = serializers.UUIDField(source='measurement.uuid')
-    quantity = serializers.FloatField()
-
-    def create(self, validated_data):
-        user = self.context['request'].user
-        create_model = self.context['view'].model
+    measurement_uuid = serializers.UUIDField(source='measurement.uuid', required=False)
+    quantity = serializers.FloatField(required=False)
```

Comment:
Spend ten minutes and make sure that this is correct, should a composition required a quantity to be passed?